app/views/account/contacts/index.html.slim

```diff
@@ -0,0 +1,41 @@
+.container
+  .row
+    .col-sm
+      h1#contact_header Contacts from users
+      = paginate @contacts
+      = search_form_for @q, url: account_contacts_path do |f|
+        = f.label :name_cont
+        = f.search_field :name_cont
+        = f.label :email_cont
+        = f.text_field :email_cont
+        = f.label :phone_cont
+        = f.text_field :phone_cont
+        = f.submit "Search", class: 'btn btn-primary'
+      .error_message
+      table.table.table-striped.table-hover#contacts_table
+        th
```

Comment:
bad html
where is TR?
read about proper table/thead/tbody/tr/td/th html

Reply:
Yes, it's just silly mistake. Fixed